Added `accentColor` and `description` parameters to `/authentication/setup` route

aileen opened the pull request:
refs https://github.com/TryGhost/Team/issues/1382

- Added two possible new setup values: `accentColor` and `description` to define the brand colour and site description on initial setup
- Updated tests to reflect those changes
- Only the params when passed and fall back to default site description

aileen committed:
Added `accentColor` and `description` parameters to `/authentication/setup` route

refs https://github.com/TryGhost/Team/issues/1382

- Added two possible new setup values: `accentColor` and `description` to define the brand colour and site description on initial setup
- Updated tests to reflect those changes
- Only the params when passed and fall back to default site description

tpatel commented on core/server/services/auth/setup.js:
```diff
@@ -77,18 +77,29 @@ async function doSettings(data, settingsAPI) {
         return user;
     }
 
+    if (!description || typeof description !== 'string') {
+        return user;
+    }
```
I think this will cause an issue: a setup call with a title but without a description will prevent the title from being set. I would remove the IF completely (unless I've missed something).

aileen replied:
This should be good now! Thanks for checking @tpatel 🤗

tpatel reviewed (changes requested):
I think an `if` might break the install process for self hosters who can't set the description in the setup screen.
<img width="511" alt="image" src="https://user-images.githubusercontent.com/494686/155532784-e9cbcfa1-4f3a-42fb-b975-7384369c9f28.png">

aileen committed:
Don't require description to setup with title

refs https://github.com/TryGhost/Team/issues/1382

- PR feedback
- Self hosters don't have the option to add a site description when setting up
- Update the logic to continue with the settings setup, even when no description is given

tpatel reviewed (approved):
Looks great 🙌
